### PR TITLE
Moving UI integration test to product-apim

### PR DESCRIPTION
**Pull request by tmkasun**
Depends on: https://github.com/wso2/carbon-apimgt/pull/6649

### CODE REVIEW

**Comment by malinthaprasan** [comment]
Should we merge this only after bumping carbon.apimgt version?

**Comment by tmkasun** [comment]
> @tmkasun Should we merge this only after bumping carbon.apimgt version?

Yes exactly, otherwise UI tests will fail.

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/UI/APIMANAGERUIIntegrationTestRunner.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        }
+        return sb.toString();
+    }
+
+    @BeforeClass(alwaysRun = true)
+    public void setEnvironment() {
+
+        try {
+            super.init();
+            publisherURLHttp = publisherUrls.getWebAppURLHttp();
```
Comment (tmkasun) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/UI/APIMANAGERUIIntegrationTestRunner.java]: Remove unused variable

Comment (tmkasun) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/UI/APIMANAGERUIIntegrationTestRunner.java]: Fixed in this PR https://github.com/wso2/product-apim/pull/5189

**Comment by tmkasun** [comment]
Review suggestions are implemented in the following PR , Hence closing this.

https://github.com/wso2/product-apim/pull/5189

**Pull request closed without merging**